### ISSUE TITLE
Multiple issues with images placement in Blog

### ISSUE THREAD

**Issue by qumuq-til** [devlom/grav-theme-hola#3]
- Vertically oriented images are not being centered inside a blog page.
- Vertically oriented images that follow one another overlap and one image covers a portion of the other.
- Text that is supposed to precede those vertical images sometimes bleeds in between them when the viewport size is intermediate between full-desktop layout and mobile layout (see examples).
![center-overlap](https://user-images.githubusercontent.com/43707841/47962769-aa015200-e032-11e8-9206-22949be7286a.png)
![bleed-in](https://user-images.githubusercontent.com/43707841/47962770-aa015200-e032-11e8-918d-30428bc1e6d2.png)
![source](https://user-images.githubusercontent.com/43707841/47962771-aa015200-e032-11e8-8fc5-79d430e06423.png)

**Comment by Dzi0nek** [comment]
To be consistant with the design and the text around img's that are inserted will not be aligned to center because the text isn't. We will provide a fix for the img's to not ovelap each other.

**Comment by qumuq-til** [comment]
I beg to differ regarding the design, because you could say that landscape images in blog examples are indeed centered, page-wise. Leaving vertically aligned images stick to the left makes for a messy look that cannot be avoided. You can try that yourself and see if there's a better solution for vertical images. Maybe a selectable option to center them will be nice.

The images do get centered if the viewport is made a little narrower as seen in the examples above.

**Comment by Dzi0nek** [comment]
The only workaround for now is to use an image in html tags like this:
`<div class="align-center" markdown="1">![](tree.jpg)</div>`

**Comment by qumuq-til** [comment]
Yep, that should work. You may also consider just stretching the vertical image in the theme to fill the entire horizontal space, i.e. making one huge vertical image.

**Comment by Dzi0nek** [comment]
We could but it usually makes page very long to scroll if couple of the image's are there and we would like to avoid that.

**Comment by qumuq-til** [comment]
Yes, that's true, I agree. But how about two stacked together? Take a look how it's done in Kalium. That approach does require identical ratios but works great.

![kalium-two-column-image](https://user-images.githubusercontent.com/43707841/47995232-a7bbf800-e105-11e8-8453-36cf6a5cbca7.png)

**Comment by Dzi0nek** [comment]
Yes but this is different situation, we're talking about image grid collection inside blog post.

**Comment by qumuq-til** [comment]
Not sure I understand what you mean, but the screenshot did come from a blog page in Kalium.
Edit: i.e. not the Portfolio module.

**Comment by Dzi0nek** [comment]
I mean you have to know before that you would like to do such image placement and provide specific classes for it. We did not changed styles that was provided by Styleshout.

**Comment by qumuq-til** [comment]
I see. With all due respect I think my reasoning is sound on this one. Design-wise something has to be done about vertical images so that they don't look out of place. Centering an image is one solution and making a grid with two stacked side by side is a good continuation of that. Just shifting a vertical image to the left completely breaks the elegance of the theme. Having users to provide all images in landscape orientation severely limits the theme's appeal.

**Comment by Dzi0nek** [comment]
I understand your point but at the moment we were not thinking about adding new features to the theme we only converted HTML theme to grav and if some functionality was missing in html version it will be missing in hola at the moment. Maybe in future we'll add new features.

**Comment by qumuq-til** [comment]
Sorry, my comment was misplaced.
